### ISSUE TITLE
Alt text

### ISSUE THREAD

**Issue by farishkash** [jrombouts31/prj-rev-bwfs-tea-cozy#3]
https://github.com/jrombouts31/prj-rev-bwfs-tea-cozy/blob/fecc89c09ac808e67f4db08325835a2111812435/Tea%20Cozy/index.html#L39

Alt text should be used on every image, this helps the visually impaired through their use of screen readers.  Also, the site will not pass HTML validation without it.
